src/core/expr/fexpr_fillna.cc

```diff
@@ -183,27 +185,7 @@ class FExpr_FillNA : public FExpr_Func {
 
       return wf;
     }
-
-
-    static Column make_isna_col(Column&& col) {
-      switch (col.stype()) {
-        case SType::VOID:    return Const_ColumnImpl::make_bool_column(col.nrows(), true);
-        case SType::BOOL:
-        case SType::INT8:    return Column(new Isna_ColumnImpl<int8_t>(std::move(col)));
-        case SType::INT16:   return Column(new Isna_ColumnImpl<int16_t>(std::move(col)));
-        case SType::DATE32:
-        case SType::INT32:   return Column(new Isna_ColumnImpl<int32_t>(std::move(col)));
-        case SType::TIME64:
-        case SType::INT64:   return Column(new Isna_ColumnImpl<int64_t>(std::move(col)));
-        case SType::FLOAT32: return Column(new Isna_ColumnImpl<float>(std::move(col)));
-        case SType::FLOAT64: return Column(new Isna_ColumnImpl<double>(std::move(col)));
-        case SType::STR32:
-        case SType::STR64:   return Column(new Isna_ColumnImpl<CString>(std::move(col)));
-        default: throw RuntimeError();
-      }
-    }
-
-};
+ };
```

Comment:
Strange indent too... Is it possible to set up your editor, so that it doesn't introduce these indentation changes?